SPECS/grub2/grub2.spec

```diff
@@ -148,6 +148,7 @@ the grub2-mkconfig flow on AzureLinux
 %package configuration
 Summary:        Location for local grub configurations
 Group:          System Environment/Base
+Requires:       %{name} >= 2.06-12
```

Comment:
Is this backwards? This means that if we install the conf files we must have full grub right?

Reply:
This becomes a non-issue now that we are targeting 3.0. All 3.0 images will ship with >= grub-2.06-12

Reply:
Actually, I think we still want the reverse of this (requires from grub -> grub-config) to block users from downgrading to a version of grub which packages the config dirs the old way.  That way, 3.0 is resilient to the unexpected case where they downgrade grub.

Reply:
Sorry for the confusion, I meant he entire requires clause seems backwards, not just the version. Shouldn't `grub2` have the requires on `grub2-configuration`?

Reply:
Yes, see current change: https://github.com/microsoft/CBL-Mariner/pull/6497/files#diff-9dd7fef1e850ad265fea94cbe5a86e163172ee833f8ec954d946d57b0ca44ce4R63